tests/unit/query/odata-adapter-test.js

```diff
@@ -761,6 +761,17 @@ test('adapter | odata | true predicate | details predicate', function (assert) {
   runTest(assert, builder, 'EmberFlexberryDummyComments', `$filter=UserVotes/all(f:true)&$select=__PrimaryKey`);
 });
 
+test('adapter | odata | true predicate | details predicate', function (assert) {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | odata | true predicate | details predicate any', function (assert) {
```